daprdocs/content/en/developing-applications/local-development/multi-app-dapr-run/multi-app-template.md

```diff
@@ -0,0 +1,90 @@
+---
+type: docs
+title: Multi-app template file
+linkTitle: Multi-app Run template
+weight: 2000
+description: Unpack the multi-app template file and its variables
+---
+
+{{% alert title="Note" color="primary" %}}
+ Multi-app `dapr run -f` is currently a preview feature only supported in Linux/MacOS. 
+{{% /alert %}}
+
+The multi-app template file is a single YAML configuration file that you can use to configure multiple applications alongside a Dapr sidecar. Execute the following command for Dapr to parse the multi-app template file, named `dapr.yaml` by default:
+
+```cmd
+dapr run -f
+```
+
+To name the multi-app template file something other than `dapr.yaml`, run:
+
+```cmd
+dapr run -f ./<your-preferred-file-name>.yaml
+```
+
+The multi-app template file can include any of the following parameters. 
+
+```yaml
+version: 1
+common: # optional section for variables shared across apps
+  resourcesPath: ./app/components # any dapr resources to be shared across apps
+  env:  # any environment variable shared across apps
+    - DEBUG: true
+apps:
+  - appID: webapp # required
+    appDirPath: .dapr/webapp/ # required
+    resourcesPath: .dapr/resources # (optional) can be default by convention
+    configFilePath: .dapr/config.yaml # (optional) can be default by convention too, ignore if file is not found.
+    appProtocol: HTTP
+    appPort: 8080
+    appHealthCheckPath: "/healthz" # All _ converted to - for all properties defined under daprd section
+    command: ["python3" "app.py"]
+  - appID: backend
+    appDirPath: .dapr/backend/
+    appProtocol: GRPC
+    appPort: 3000
+    unixDomainSocket: "/tmp/test-socket"
+    env:
+      - DEBUG: false
+    command: ["./backend"]
+```
+
+## Parameters
+
+
+| Parameter                | Required | Details | Example |
+|--------------------------|:--------:|--------|---------|
+| `appID`                  | Y        | Your application's app ID | `webapp`, `backend` |
+| `appDirPath`             | Y        | Path to the your application | `./webapp/`, `./backend/` |
+| `resourcesPath`          | N        | Path to your Dapr resources. Can be default by convention; ignore if directory isn't found | `./app/components`, `./webapp/components` |
+| `configFilePath`         | N        | Path to your application's configuration file | `./webapp/config.yaml` |
+| `appProtocol`            | N        | Application protocol | `HTTP`, `GRPC` |
```

Comment:
```suggestion
| `appProtocol`            | N        | The port your application is listening on | `HTTP`, `GRPC` |
```
Suggest that for many of the comments here you can take the same description as the table here https://docs.dapr.io/reference/cli/dapr-run/. I would also reference this table since they should be the same.